index.html

```diff
@@ -0,0 +1,484 @@
+                 <span class="portfolio-overlay-filter"> </span>
+                 <div class="portfolio-overlay-button"><a href="">+</a></div>
+               </div>
+             </div>
+           </div>
+            <figcaption>
+              <p><strong class="f-bold">Facebook thumbs</strong></p>
+              <p>Icons design, brand & logotypes</p>
+            </figcaption>
+          </figure>
+        </div>
+      </article>
+      <article class="b-black" id="about">
+        <h2 class="f-bold f-large">ABOUT</h2>
+        <section class="l-flex-container">
+          <p class="l-flex-item l-about-txt">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a gallery of type and scrambled it to make a type specimen book.</p>
+          <p class="l-flex-item l-about-txt">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a gallery of type and scrambled it to make a type specimen book.</p>
+        </section>
+        <section>
+          <h3 class="f-bold h3-lines">SKILLS</h3>
+          <p class="s-progress-paragraph">PHOTOSHOP</p><progress min="0" max="100" value="80"></progress></p>
+          <p class="s-progress-paragraph">HTML/CSS</p><progress min="0" max="100" value="90"></progress></p>
+          <p class="s-progress-paragraph">ILLUSTRATOR</p><progress min="0" max="100" value="70"></progress></p>
+          <p class="s-progress-paragraph">PHP</p><progress min="0" max="100" value="60"></progress></p>
+          <p class="s-progress-paragraph">WORDPRESS</p><progress min="0" max="100" value="95"></progress></p>
+        </section>
+        <section>
+          <h4 class="f-light s-progress-question-text">Do you want be one of Avengers? Join us!</h4>
+          <a href=""><button class="btn contact-btn">CONTACT US</button></a>
+        </section>
+      </article>
+      <article class="b-white b-black-white" id="team">
+        <h2 class="f-bold f-large">TEAM</h2>
+        <section>
+          <p>Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia.</p>
+        </section>
+        <section>
+          <ul class="l-flex-container">
+            <li class="l-flex-item l-team">
+              <p class="f-bold f-team"><strong>JONATHAN DOE</strong></p>
+              <p class="f-team-b">CEO, UI&UX designer</p>
+              <figure>
+                <img src="/img/jonathan.png" alt="Jonathan Doe Photo" class="l-flex-child l-child-m">
+                <figcaption>
+                  <a href="" class="icon-facebook2 icon-social"></a>
```

Comment:
This is another nitpick, but for the sake of readability classes should go general to specific (especially on a stack of elements like this); so, `icon-social icon-facebook2`. That makes it easier to scan the class list and see the differences.